modules/apps/frontend-js/frontend-js-components-sample-web/src/main/resources/META-INF/resources/partials/feature_indicator.jsp

```diff
@@ -35,64 +35,34 @@
 		<clay:col>
 			<h3>Beta Interactive</h3>
 
-			<react:component
-				module="{FeatureIndicator} from frontend-js-components-web"
-				props='<%=
-					HashMapBuilder.<String, Object>put(
-						"interactive", "true"
-					).put(
-						"learnResourceContext", LearnMessageUtil.getReactDataJSONObject("frontend-js-components-web")
-					).put(
-						"type", "beta"
-					).build()
-				%>'
+			<liferay-frontend:feature-indicator
```

Comment:
The title at line 30 needs to be updated to indicate this is using the taglib not direct instantiation of the React component.